experiments/compositions/expanders/helm-expander/Makefile

```diff
@@ -146,21 +146,24 @@ docker-run: docker-build
 create-kind:
 	kind delete clusters ${KIND_CLUSTER} || true
 	kind create cluster --name ${KIND_CLUSTER}
-	kind load docker-image ${EXPANDER_IMG} --name ${KIND_CLUSTER}
 
 .PHONY: deploy-kind
-deploy-kind: kustomize manifests docker-build create-kind
-	$(KUBECTL) --context kind-${KIND_CLUSTER} create namespace composition-system
+deploy-kind: kustomize manifests docker-build
+	kind load docker-image ${EXPANDER_IMG} --name ${KIND_CLUSTER}
+	# this is ok to fail for existing kind cluster
+	$(KUBECTL) --context kind-${KIND_CLUSTER} create namespace composition-system || true
```

Comment:
Nit: I usually do `kubectl create --dry-run=client -oyaml ns foo | kubectl apply -f -` to get around this.  This works though!